### DISCUSSION TITLE
Smooth resetTurntableRotation()

### DISCUSSION THREAD

**Discussion by Aztec91** [google/model-viewer#2362] [Q&A]
Hey Guys,

I got another question: is there an easy way to reset the TurntableRotation in a smooth way?

Like the interpolation between 2 camera-orbits?

Example: https://commacross.de/3DAR/motor2.html

the second dot triggers the auto-rotation function. 
When I press the third dot, I get a "hard-cut" with the camera. Is there a way to make this smooth?

kind regards

**Comment by elalish** [comment]
That's a great demo! You should share this in show & tell when you feel it's ready. I see what you mean and I'd recommend two options. Either skip `auto-rotate` and do the rotation by setting `camera-orbit` in a `requestAnimationFrame()`, or just stop the rotation instead of resetting the turntable and add the `turntableRotation` offset to your following `camera-orbit` commands.

**Reply by Aztec91** [reply]
Thank you very much.

You have been a hughe help getting there.

I will consider posting it in show&tell

Regards
Aztec
